stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/FileAttachmentContent.kt

```diff
@@ -71,10 +73,18 @@ public fun FileAttachmentContent(
  */
 @Composable
 public fun FileAttachmentItem(attachment: Attachment) {
+    val context = LocalContext.current
+
     Surface(
         modifier = Modifier
             .padding(2.dp)
-            .fillMaxWidth(),
+            .fillMaxWidth()
+            .clickable {
+                AttachmentPreviewHandler
+                    .defaultAttachmentHandlers(context)
```

Comment:
We're creatign these handlers every time we click something.

Can we move this to ChatTheme?